src/core/PresetPreviewPlayHandle.cpp

```diff
@@ -144,22 +144,34 @@ PresetPreviewPlayHandle::PresetPreviewPlayHandle( const QString & _preset_file,
 	}
 	else
```

Comment:
I don't like that we have that `_load_by_plugin` check, a better architecture for this would be to have two different constructor instead of a bif if..else that makes the `dataFile` useless in one of the branches.

However I am glad that this is much better than the previous implementation, good job! :+1: